pkg/utils/gzip.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"bufio"
+	"compress/gzip"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+func GzipCompress(file string) (string, error) {
+	fileData, err := os.Open(file)
+
+	if err != nil {
+		return "", err
+	}
+
+	read := bufio.NewReader(fileData)
+
+	data, err := ioutil.ReadAll(read)
+
+	if err != nil {
+		return "", err
+	}
+
+	newFile := strings.Replace(file, ".txt", ".gz", -1)
```

Comment:
any reason not to just concat the `.gz` onto the end of the file?

```suggestion
	newFile := file + ".gz"
```

Reply:
I see no reason :)